### PR TITLE
[FSAR-269] Multipage guide hero

### PR DESCRIPTION
**Pull request by c-ancia**
Please provide a thorough description of your component and its usage with screenshots.

## Checklist before opening a PR

Before opening a PR, please make sure:

- [ ] your work is in a branch based on and synced with the develop branch.
- [ ] your scss and twig files are imported in the javascript file. Without this, Webpack won't be able to compile them.
- [ ] your newComponent is imported in the index.js file.
- [ ] your folder hierarchy follows the proper structure.
- [ ] your code follows the coding standards and naming conventions defined.
- [ ] your code has been tested as AA accessibility compliant to the WCAG 2.1 AA standard.
- [ ] your code has been tested in all the browsers listed in the GDS browser's list, including IE11.
- [ ] your code has been tested in all the different screensizes defined.

### CODE REVIEW

File: src/components/form/Dropdown/dropdown.html.twig [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <p class="dropdown__description">{{ description }}</p>
   {% endif %}
   <div class="dropdown__field">
-    <select name="{{ name }}" id={{ id }} class="dropdown__select" 'data-drupal-states="{{ states ?  states|json_encode : '' }}"'>
+    <select name="{{ name }}" id={{ id }} class="dropdown__select" data-drupal-states="{{ states ? states : '' }}" >
```
Comment (EmilyHazlehurst) [inline src/components/form/Dropdown/dropdown.html.twig:13]: This is the tiniest thing, but there probably shouldn't be a space before the >.

**Review by EmilyHazlehurst — APPROVED** [review]

File: src/components/form/Dropdown/dropdown.html.twig (hunk 1 above)
Comment (c-ancia) [reply src/components/form/Dropdown/dropdown.html.twig:13]: oh, I can explain why. Drupal is a bit acting up with the drupal-states 😭 when it reads the json, it says something like "unexpected end of JSON file" and it seems to take the > in consideration :'( adding the space seemed to help, but TBH, the problem is not fully resolved yet.

Comment (EmilyHazlehurst) [reply src/components/form/Dropdown/dropdown.html.twig:13]: Is that because the id should be `id="{{ id }}"`? My IDE complains without the "".

Comment (c-ancia) [reply src/components/form/Dropdown/dropdown.html.twig]: I finally fixed the issue so I'm merging this :) I removed the spacing. Thank you for your review! :)

**Pull request merged**